duplicate.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Using latent semantic indexing and similarity matrix
+
+import logging
+from collections import defaultdict
+
+import nltk
+from gensim import models, similarities
+from gensim.corpora import Dictionary
+from nltk.corpus import stopwords
+from nltk.stem.porter import PorterStemmer
+
+from bugbug import bugzilla
+
+logging.basicConfig(
+    format="%(asctime)s : %(levelname)s : %(message)s", level=logging.INFO
+)
+
+nltk.download("stopwords")
+
+# A map from bug id to its duplicate ids
+all_ids = defaultdict(int)
+duplicates = defaultdict(set)
+
+for bug in bugzilla.get_bugs():
+    all_ids[bug["id"]] = 1
```

Comment:
Instead of this, you can just do:
```Python
all_ids = set(bug for bug in bugzilla.get_bugs())
```

But, given that we want to ignore bugs from some specific reporters, you can do (like in bugbug/models/duplicate.py):
```Python
        all_ids = set(
            bug["id"]
            for bug in bugzilla.get_bugs()
            if bug["creator"] not in REPORTERS_TO_IGNORE
            and "dupeme" not in bug["keywords"]
        )
```